bindings/typescript/src/index.ts

```diff
@@ -0,0 +1,42 @@
+import { Tiktoken } from "./wrap";
+
+async function main() {
```

Comment:
I think the goal of these different language binding packages should be to publish them on different package repositories for consumption (npm, pypi, etc).

The way this package is created does not support this. We need to (1) have an entry-point file that exports types (2) have a build step that compiles to .js & .d.ts, and (3) have tests that utilize this built package.